data/settings.js

```diff
@@ -6962,14 +6962,20 @@ format), save the message to the detail mailbox.`
 
 	lmtp_user_concurrency_limit: {
 		tags: [ 'lmtp', 'user_concurrency_limits' ],
+		changed: {
```

Comment:
Should also add to 2.4-to-2.4.x.md upgrade page. And to v3.1 upgrade page in Pro docs.